NEWS.md

```diff
@@ -1,8 +1,12 @@
 # testthat 0.9.1.9000
 
+* test (test_dir, test_file, test_package, test_check) functions now return a
```

Comment:
Yes, but it still returns invisibly the test results. I guess there's no harm in that ?